plugins/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "appium-inspector-plugin",
+  "version": "2024.12.1",
```

Comment:
how this version is supposed to be bumped/synchronized with main inspector version?

Reply:
Currently we are manually updating the package.json. If are still sticking to manual then we need to update in both the places. 

@eglitise can add more, I have not done any inspector release before.

Reply:
We might need to add a release script to bump both versions simultaneously. Currently the main version bump is done via `npm version <version>`, which edits `package.json`, adds the version tag, then commits and pushes the changes.

Reply:
Yea, I've been working slowly for this PR but thought to prepare a script to update the `version` in `plugins/package.json` a part of this pr.